src/librustdoc/html/static/main.js

```diff
@@ -2374,6 +2355,30 @@ if (!DOMTokenList.prototype.remove) {
         };
     });
 
+    // In the search display, allows to switch between tabs.
+    function printTab(nb) {
```

Comment:
Why was this function moved? It doesn't seem to have anything to do with this PR, and it breaks the ability to do `git blame` analysis on it. (And probably will create a merge conflict with https://github.com/rust-lang/rust/pull/58364 on top of that.)

Reply:
It was between functions that should be along each other. I can move it back if you want?

Reply:
It's fine; i won't let that hold up the PR.